src/contrib/mlir/backend/pass/affine_lowerer.cpp

```diff
@@ -249,6 +248,7 @@ namespace
         inline size_t insertAttrs(opAttrs attrs);
 
         MemoryAnalysis* getMemAnalysis() const { return m_memAnalysis; }
+
```

Comment:
extra space.

Reply:
Let me know if it's OK now. It seems a clang-format thing.